share-extension/ShareViewController.m

```diff
@@ -27,10 +27,13 @@
 #import "UploadViewController.h"
 
 #import <MobileCoreServices/MobileCoreServices.h>
+#import <ImageIO/ImageIO.h>
+
 #define kRestVersion @"v1-alpha3"
 #define kRestContextName @"rest"
 #define kPortalContainerName @"portal"
 #define kMaxSize    10000000
+#define kJPEGCompressionLevel 1.0
```

Comment:
Leave a comment to indicate the possible quality / compression levels, e.g. 0.0 min compression - 1.0 max compression